### PR TITLE
Travis CI

### PR DESCRIPTION
**Pull request by smoothdeveloper**
#1417

Right now 540 tests are passing, 63 failing.

You can check https://travis-ci.org/smoothdeveloper/VisualFSharpPowerTools

### CODE REVIEW

File: build.fsx [hunk 1]
```diff
@@ -114,6 +119,11 @@ Target "Build" (fun _ ->
     |> ignore
 )
 
+Target "BuildFSharpEditingOnly" (fun _ ->
```
Comment (dungpa) [inline build.fsx]: This target doesn't seem to be used anymore.

File: build.fsx [hunk 2]
```diff
@@ -13,6 +14,10 @@ open System.IO
 open System.Xml
 open Fake.Testing
 
+type RootFolder = FSharp.Management.FileSystem< const(__SOURCE_DIRECTORY__) >
+
+let (</>) a b = Path.Combine (a, b)
```
Comment (dungpa) [inline build.fsx]: I thought this operator is available in FAKE, no?

**Comment by dungpa** [comment]
Majority of the failing tests can be fixed by changing paths in https://github.com/fsprojects/VisualFSharpPowerTools/blob/08041036cd5125eaf21f3a48ecab674c4843ebac/tests/FSharp.Editing.Tests/CodeGenerationTestInfrastructure.fs#L84 to "/File.fs".

File: build.fsx (hunk 1 above)
Comment (smoothdeveloper) [inline build.fsx]: Sorry removing that useless target now.

**Comment by smoothdeveloper** [comment]
Thanks for feedback @dungpa I propose we look at the failing test separately at later point.

**Pull request merged**